salt/modules/win_update.py

```diff
@@ -29,6 +29,7 @@
 # pylint: enable=import-error
 
 # Import salt libs
+from salt.utils.locales import sdecode
```

Comment:
This needs to be `import salt.utils.locales` if we're going to used the fully-qualified name below.